test/runner/lib/util.py

```diff
@@ -617,8 +651,8 @@ def print_message(self, message, color=None, fd=sys.stdout, truncate=False):  #
             message = message.replace(self.clear, color)
             message = '%s%s%s' % (color, message, self.clear)
 
-        if sys.version_info[0] == 2 and isinstance(message, type(u'')):
-            message = message.encode('utf-8')
+        if sys.version_info[0] == 2:
+            message = to_bytes(message)
```

Comment:
I've never seen `print(file=...)` before, should we still be printing a byte string or should it be a native string?

Reply:
This change just standardizes on the new `to_bytes` function instead of using `encode()` to do the same thing. I was trying to minimize the amount of encoding related changes in this PR that may actually change behavior, other than to fix issues I was encountering during testing.

I'd like to do more encoding cleanup in a follow up PR, including taking another look at this. I just didn't want to do it all at once.